### ISSUE TITLE
App ID with Identifier not available.

### ISSUE THREAD

**Issue by gilosborne** [jasarien/Provenance#253]
One I sign in to my developer account and try to run again I get this error attached. How do I get past this?
![screen shot 2015-12-23 at 6 14 59 pm](https://cloud.githubusercontent.com/assets/6044562/11986759/2c4008c2-a9a1-11e5-8dca-06ea78c1cc80.png)

**Comment by jasarien** [comment]
You follow the build instructions, where it explicitly instructs you to change the bundle ID string to avoid this error. https://github.com/jasarien/Provenance/wiki/Building-Provenance

**Comment by gilosborne** [comment]
I did that and I still got it. the code in the error is not changing even though I changed it in the settings. See attachments.
![screen shot 2015-12-23 at 6 53 06 pm](https://cloud.githubusercontent.com/assets/6044562/11987084/e8319f14-a9a6-11e5-9814-94c48ee5d4ce.png)
![screen shot 2015-12-23 at 6 52 53 pm](https://cloud.githubusercontent.com/assets/6044562/11987085/e90506f6-a9a6-11e5-9fdb-05cb6c94d9e1.png)
